docs/src/HowToGuides/Atmos/Microphysics.md

```diff
@@ -132,9 +284,30 @@ where:
  - ``q_{liq}`` - liquid water specific humidity,
  - ``\tau_{cond\_evap}`` - relaxation timescale (parameter in `MicrophysicsParameters` module).
 
-## Autoconversion
+### Cloud ice sublimation/resublimation
+
+Sublimation and resublimation of cloud ice is parameterized as a relaxation to equilibrium value at the current time step.
+```math
+\begin{equation}
+  \left. \frac{d \, q_{ice}}{dt} \right|_{sub, resub} = \frac{q^{eq}_{ice} - q_{ice}}{\tau_{sub\_resub}}
+\end{equation}
+```
+where:
+ - ``q^{eq}_{ice}`` - ice specific humidity in equilibrium,
+ - ``q_{ice}`` - ice specific humidity,
+ - ``\tau_{sub\_resub}`` - relaxation timescale (parameter in `MicrophysicsParameters` module).
```

Comment:
Can we make this a function, which right now just returns a parameter? I'd soon like to be able to include the relationships that Raymond Shaw showed us.

Reply:
done